tests/drivers/video/api/src/video_emul.c

```diff
@@ -157,7 +157,7 @@ ZTEST(video_common, test_video_vbuf)
 	zexpect_ok(video_set_format(rx_dev, VIDEO_EP_OUT, &fmt));
 
 	/* Allocate a buffer, assuming prj.conf gives enough memory for it */
-	vbuf = video_buffer_alloc(fmt.pitch * fmt.height, K_FOREVER);
+	vbuf = video_buffer_alloc(fmt.pitch * fmt.height, K_NO_WAIT);
```

Comment:
This change is not explained in the commit message. Also I suppose the below assertion is then not valid anymore?

Reply:
Good point thank you, I will revisit it.

Reply:
This is now its [own commit](https://github.com/zephyrproject-rtos/zephyr/compare/da560abfeea958cd2cf7fe79dd5ececf5b0f7ed8..31d459b384335f40db64ae10b97409f299924529) proposing not to wait CI timeout when running out of video memory.